cypress/integration/console/packet-broker/routing-policies.spec.js

```diff
@@ -0,0 +1,163 @@
+      .parent()
+      .within(() => {
+        cy.findByLabelText('Join accept').check()
+        cy.findByLabelText('MAC data').check()
+        cy.findByLabelText('Application data').check()
+      })
+    cy.findByRole('button', { name: 'Save default policy' }).click()
+
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.findByTestId('toast-notification')
+      .should('be.visible')
+      .findByText('Default routing policy set')
+      .should('be.visible')
+  })
+
+  it('succeeds unsetting a default routing policy', () => {
+    cy.intercept('GET', '/pba/home-networks/policies/default', {
+      fixture: 'console/packet-broker/default-policy.json',
+    })
+    cy.intercept('DELETE', '/pba/home-networks/policies/default', {})
+
+    cy.visit(`${Cypress.config('consoleRootPath')}/admin/packet-broker`)
+
+    cy.findByLabelText('Do not use a default routing policy for this network').check()
+    cy.findByRole('button', { name: 'Save default policy' }).click()
+
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.findByTestId('toast-notification')
+      .should('be.visible')
+      .findByText('Default routing policy set')
+      .should('be.visible')
+  })
+
+  it('succeeds setting individual per-network routing policy', () => {
+    cy.intercept('GET', '/pba/home-networks/policies/default', {
+      fixture: 'console/packet-broker/default-policy.json',
+    })
+    cy.intercept('GET', '/pba/home-networks/policies/19', {
+      statusCode: 404,
+      fixture: '404-body.json',
+    })
+    cy.intercept('PUT', '/pba/home-networks/policies/19', {})
+
+    cy.visit(`${Cypress.config('consoleRootPath')}/admin/packet-broker/networks/19`)
+
+    // Check routing policy form checkboxes.
+    cy.findByLabelText('Use network specific routing policy').check()
+    cy.findByText('Set routing policy towards this network')
+      .parent()
+      .within(() => {
+        cy.findByText('Uplink')
+          .parent()
+          .within(() => {
+            cy.findByLabelText('Join request').check()
+            cy.findByLabelText('MAC data').check()
+            cy.findByLabelText('Application data').check()
+            cy.findByLabelText('Signal quality information').check()
+            cy.findByLabelText('Localization information').check()
+          })
+        cy.findByText('Downlink')
+          .parent()
+          .within(() => {
+            cy.findByLabelText('Join accept').check()
+            cy.findByLabelText('MAC data').check()
+            cy.findByLabelText('Application data').check()
+          })
+        cy.findByRole('button', { name: 'Save routing policy' }).click()
+      })
+
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.findByTestId('toast-notification')
+      .should('be.visible')
+      .findByText('Routing policy set')
+      .should('be.visible')
+  })
+
+  it('succeeds unsetting individual per-network routing policy (without default policy)', function () {
```

Comment:
```suggestion
  it('succeeds unsetting individual per-network routing policy (without default policy)', () => {
```

Reply:
It needs to use standard functions due to arrow function's lexical binding of `this` (used in `L126` and `L146`).